test/unit/controllers/CustomsNotificationControllerSpec.scala

```diff
@@ -220,6 +221,26 @@ class CustomsNotificationControllerSpec extends UnitSpec with Matchers with Mock
       }
     }
 
+    "extractFunctionCode from xml payload when present" in {
+      controller.extractFunctionCode(elementToOptionalNodeSeq(ValidXML)) shouldBe None
```

Comment:
convention is to have `()` on this type of call. Applies five more times in this change.

Reply:
Ok, done.